amy/workshops/views.py

```diff
@@ -2186,11 +2187,22 @@ def get_initial(self, **kwargs):
 
         # Determine initial event in AwardForm
         if "find-training" in self.request.GET:
-            tasks = Person.objects.get(
-                pk=self.request.GET["person"]
-            ).get_training_tasks()
-            if tasks.count() == 1:
-                initial.update({"event": tasks[0].event})
+            initial.update(
+                {
+                    "badge": Badge.objects.get(name="instructor"),
+                }
+            )
```

Comment:
Probably can be simplified:
```suggestion
            initial["badge"] = Badge.objects.get(name="instructor")
```